### PR TITLE
added: filters to product page

### PR DESCRIPTION
**Pull request by harshalini**
Added filter by price, sort by price, sort by category, sort by ratings and search bar to products page.
Check it here->
https://modest-stonebraker-d174d5.netlify.app/products.html

Screenshot (312)

### CODE REVIEW

**Comment by bhargaviRdy** [comment]
the code looks good and the color you choosed is awesome. One small suggestion would be is to make the entire site responsive. It looks even more prettier than now.

**Review by bhargaviRdy — APPROVED** [review]

File: products.css [hunk 1]
```diff
@@ -18,9 +18,68 @@
...
+.sidebar span {
+    font-weight: 500;
+    margin-left: 1rem;
+    border-bottom: solid 3px var(--orange);
+    padding-bottom: 2px;
+}
+select {
+    padding: 3px 1rem;
+    margin: auto;
+    font-size: 16px;
```
Comment (t007rushi) [inline products.css]: use rem instead

File: products.css [hunk 2]
```diff
@@ -18,9 +18,68 @@
...
+    color: var(--orange);
+    cursor: pointer;
+}
+.sidebar span {
+    font-weight: 500;
+    margin-left: 1rem;
+    border-bottom: solid 3px var(--orange);
+    padding-bottom: 2px;
+}
+select {
```
Comment (t007rushi) [inline products.css]: use className instead direct tag for good Practices

**Review by t007rushi — APPROVED** [review]
else everything looks clean code practices

File: products.css (hunk 1 above)
Comment (harshalini) [reply products.css]: Yes, it was a silly mistake😅

File: products.css (hunk 2 above)
Comment (harshalini) [reply products.css]: Okay, thank you for suggesting😊

**Comment by harshalini** [comment]
> the code looks good and the color you choosed is awesome. One small suggestion would be is to make the entire site responsive. It looks even more prettier than now.

Thank you! Once the project is done, I am going to make it responsive

**Pull request merged**